### PR TITLE
Corrected default values of checksum and analyze in Lightning.

### PR DESCRIPTION
**Pull request by hfxsd**
### What is changed, added or deleted? (Required)

Corrected default values of checksum and analyze in Lightning.

- checksum should be "required" by default
- analyze should be "optional" by default

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-lightning/tidb-lightning-configuration.md [hunk 1]
```diff
@@ -240,17 +240,28 @@ max-allowed-packet = 67_108_864
...
+
 # 如果设置为 true，会在导入过程结束时对整个 TiKV 集群执行一次 full Compact。
 # 默认值为 false。
 compact = false
+
 # 如果设置为 true，会对所有表逐个执行 `ANALYZE TABLE <table>` 操作。
-analyze = true
+# 如果设置为 optional，会对所有表逐个执行 `ANALYZE TABLE <table>` 操作，但忽略执行结果的中的错误。
+# 默认值为 optional。
+analyze = optional
```
Comment (gozssky) [inline tidb-lightning/tidb-lightning-configuration.md:264]: analyze 应该也支持像 checksum 那样的 off 和 required ?

**Review by gozssky — APPROVED** [review]

File: tidb-lightning/tidb-lightning-configuration.md [hunk 2]
```diff
@@ -240,17 +240,31 @@ max-allowed-packet = 67_108_864
...
-# 如果设置为 true，会对所有表逐个执行 `ADMIN CHECKSUM TABLE <table>` 操作
-# 来验证数据的完整性。
-checksum = true
+# 设置对所有表逐个执行 `ADMIN CHECKSUM TABLE <table>` 操作的行为，用于验证数据的完整性。
+# 有以下选项:
+# - "off"：不执行 checksum。
+# - "optional"：执行 admin checksum，如果 checksum 失败，则忽略出现的错误。
+# - "required"：执行 admin checksum，如果 checksum 失败，TiDB Lightning 退出。
+# 默认值为 "required"。从 v4.0.8 开始，checksum 的默认值由此前的 "true" 改为 "required"。
+# 说明：为了保持兼容性，bool 值`true` 和 `false` 仍然支持。 其中 `true` 等同于 "required"，`false` 等于 "off"。
```
Comment (gozssky) [inline tidb-lightning/tidb-lightning-configuration.md:249]: optional, required, true, false 这几个词的引号要不要统一下？

File: tidb-lightning/tidb-lightning-configuration.md [hunk 3]
```diff
@@ -240,17 +240,31 @@ max-allowed-packet = 67_108_864
...
 compact = false
-# 如果设置为 true，会对所有表逐个执行 `ANALYZE TABLE <table>` 操作。
-analyze = true
+
+# 设置对所有表逐个执行 `ANALYZE TABLE <table>` 操作的行为。
+# 有以下选项:
+# - "off"：不执行 analyze。
+# - "optional"：执行 analyze，如果 analyze 失败，则忽略出现的错误。
+# - "required"：执行 analyze，如果 analyze 失败，TiDB Lightning 退出。
+# 默认值为 optional。从 v4.0.8 开始，analyze 的默认值由此前的 true 改为 optional。
```
Comment (Liuxiaozhen12) [inline tidb-lightning/tidb-lightning-configuration.md:265]:
```suggestion
# 默认值为 "optional"。从 v4.0.8 开始，analyze 的默认值由此前的 "true" 改为 "optional"。
```

**Review by Liuxiaozhen12 — APPROVED** [review]
Rest LGTM

File: tidb-lightning/tidb-lightning-configuration.md (hunk 2 above)
Comment (hfxsd) [reply tidb-lightning/tidb-lightning-configuration.md:249]: 好的。都加上双引号吧，保持统一。

**Review by TomShawn — APPROVED** [review]

**Pull request merged**